src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -117,6 +117,20 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    */
   _mediaErrorRecoveryAttempts: number = 0;
 
+  /**
+   * The last time detach occurred
+   * @type {number}
```

Comment:
```suggestion
   * @type {?number}
```
but see my comment below, that the default could be `0` and type is `number`